resources/benchmark-runner.js

```diff
@@ -95,33 +61,12 @@ BenchmarkRunner.prototype._appendFrame = function (src) {
     return frame;
 }
 
-BenchmarkRunner.prototype._waitAndWarmUp = function () {
-    var startTime = Date.now();
-
-    function Fibonacci(n) {
-        if (Date.now() - startTime > 100)
-            return;
-        if (n <= 0)
-            return 0;
-        else if (n == 1)
-            return 1;
-        return Fibonacci(n - 2) + Fibonacci(n - 1);
-    }
-
-    var promise = new SimplePromise;
-    setTimeout(function () {
-        Fibonacci(100);
-        promise.resolve();
-    }, 200);
-    return promise;
-}
-
 BenchmarkRunner.prototype._writeMark = function(name) {
     if (window.performance && window.performance.mark)
         window.performance.mark(name);
 }
 
-// This function ought be as simple as possible. Don't even use SimplePromise.
+// This function ought be as simple as possible. Don't even use Promise.
```

Comment:
(random comment, the setTimeout call there isn't "simple". An implementation may or may not call rAF before the timeout runs. And rAF can be rather heavy operation. So the results can be a bit surprising.  But not about this patch :) )

Reply:
Yeah, I think we want to refine this stuff in v3. I'm observing odd behavior for https://github.com/WebKit/Speedometer/pull/11 where async time sometimes disappears.